src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1146,10 +1147,12 @@ protected override void GetItem(string path)
                         }
                     }
                 }
+#endif
 
                 FileSystemInfo result = GetFileSystemItem(path, ref isContainer, false);
                 if (result != null)
                 {
+#if !UNIX
```

Comment:
The `WriteItemObject` code applies to both UNIX and !UNIX.  If I move that bit of code up, I'd need more `#ifdefs` making it less readable